modules/core/include/opencv2/core/hal/intrin_rvv.hpp

```diff
@@ -1949,7 +1951,7 @@ inline void v_pack_store(_Tp* ptr, const _wTpvec& a) \
     _wTp CV_DECL_ALIGNED(32) arr[_Tpvec::nlanes] = {0}; \
```

Comment:
Memory usage looks very strange here. There are 32 vector registers and expect that universal intrisnic can be implemented without memory transactions.

Reply:
I removed all the redundant CV_DECL_ALIGNED statements. They seem unnecessary. Now all the tests can still pass.

Reply:
Just removal of `CV_DECL_ALIGNED(32)` modifier is a bad idea.
I believe you have asked to remove usage of memory buffer.

BTW, modern compilers are smart enough to eliminate unnecessary memory operations. Check generated assembly code of some test function to see result.